backend/src/main/java/gov/cdc/usds/simplereport/service/AddressValidationService.java

```diff
@@ -144,6 +145,7 @@ public ZoneId getZoneIdByLookup(Lookup lookup) {
       return null;
     }
 
-    return ZoneId.of("US/" + timezoneInfo.timezoneCommonName);
```

Comment:
For a valid Puerto Rico address, we were getting back `Atlantic` for `timezoneInfo.timezoneCommonName`.
If you see the list [here](https://howtodoinjava.com/java/date-time/supported-zone-ids-offsets/), `US/Atlantic` doesn't exist and was throwing an exception.

Reply:
I think using the `ZoneOffset` might run into issues with whether `ZoneId.of()` is returning the correct zone since it would change based on whether the zone obeys daylight savings time or not. The slight inaccuracy might not matter, but the original issue that forced us to start handling time zones in bulk upload was a STLT having issues with some of their data being marked as an incorrect day due to time wackiness. Wondering if it would be better in this case to just specifically handle the Atlantic time zone and still use the existing `US/commonName` to handle the other time zones since Smartystreets is still able to pinpoint what time zone the address is in compared to the possible ambiguity with `utcOffset`? I wish Smartystreets simply returned the official time zone rather than just the common name 😩